win32_build.sh

```diff
@@ -73,7 +73,8 @@ if [[ -n $CLEAN_BUILD ]]; then
     rm -rf $BUILD_DIR
 fi
 
-if [[ ! -d $DEPS_DIR ]]; then
+# If libdokan.a doesn't exist, build the dependencies
```

Comment:
I'll add an inline comment stating that libdokan should be built last since we may add other dependencies.

Reply:
An alternative would be to just have a file called ".completed" or something like that in the "build.deps" dir so that we don't tie ourselves to one specific dependency. Anyway, that's a really minor detail.

Reply:
I had the same thought as well, just didn't take the time to do it. That would be a better approach.